lib/qif/writer.rb

```diff
@@ -61,11 +62,17 @@ def initialize(io, type = 'Bank', format = 'dd/mm/yyyy')
     
     # Add a transaction for writing
     def <<(transaction)
-      @transactions << transaction
+      case transaction.class.to_s
```

Comment:
Yes I agree now. The QIF format seems very loose so good to let the developer define the format as per their use case. So we now have multiple Accounts and Transactions. No need to create any relationship between them in this gem.

 I have just submitted a PR to @davidsklar that handles writing multiple accounts. https://github.com/davidsklar/Qif/pull/3

@davidsklar Can you merge please.